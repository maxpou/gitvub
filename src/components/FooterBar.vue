<template>
  <footer class="footer">
    <div class="container">
      <span class="text-muted small"><a href="https://github.com">GitHub</a> + <a href="https://vuejs.org/">VueJS</a> = GitVub.</span>
      <span class="me text-muted text-right">@maxpou</span>
    </div>
  </footer>
</template>

<style scoped>
.footer {
  position: absolute;
  bottom: 0;
  width: 100%;
  height: 40px;
  line-height: 40px;
  background-color: #f5f5f5;
}
.text-muted > a {
  color: inherit
}
.me {
  opacity: 0.65;
  float: right;
}
</style>
