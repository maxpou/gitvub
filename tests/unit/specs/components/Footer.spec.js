import { mount } from 'vue-test-utils'
import FooterBar from '@/components/FooterBar.vue'

describe('FooterBar.vue', () => {
  it('test initial rendering', () => {
    const wrapper = mount(FooterBar)
    const template = wrapper.html()
    expect(template).toMatchSnapshot()
  })
})
