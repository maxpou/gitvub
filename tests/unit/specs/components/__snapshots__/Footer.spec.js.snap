// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterBar.vue test initial rendering 1`] = `
"<footer class=\\"footer\\"><div class=\\"container\\"><span class=\\"text-muted small\\"><a href=\\"https://github.com\\">GitHub</a> + <a href=\\"https://vuejs.org/\\">VueJS</a> =
      <router-link to=\\"[object Object]\\">GitVub</router-link>.
    </span> <span class=\\"me text-muted text-right\\">@maxpou</span></div></footer>"
`;
